collect_app/src/main/java/org/odk/collect/android/utilities/WidgetAppearanceUtils.java

```diff
@@ -82,7 +90,36 @@
     public static final String URL                      = "url";
     public static final String RATING                   = "rating";
 
-    private WidgetAppearanceUtils() {
+    public void setCameraIdIfNeeded(FormEntryPrompt prompt, IntentIntegrator intent) {
```

Comment:
I didn't mean moving both those methods here. I expected just a small isFrontCameraAppearance() method that would return true/false based on appearance. Just something like isFlexAppearance or other methods below.

Reply:
Oh alright. This makes more sense. Sorry for misinterpreting.